### PR TITLE
Add support for more than one destination

### PR DESCRIPTION
**Pull request by davorb**

### CODE REVIEW

File: lib/fortytwoish/client.rb [hunk 1]
```diff
@@ -15,7 +15,12 @@ def send
 
     private
 
-    attr_reader :message, :number
+    attr_reader :message
+
+    def numbers
+      return [@numbers] unless @numbers.kind_of? Array
+      @numbers
+    end
```
Comment (lunich) [inline lib/fortytwoish/client.rb]: Probably we can have `attr_reader :numbers` instead + do `@numbers = Array.wrap(numbers)` in the `initialize`?

**Review by lunich — CHANGES_REQUESTED** [review]

File: lib/fortytwoish/client.rb (hunk 1 above)
Comment (lunich) [reply lib/fortytwoish/client.rb]: Thanks to @jonkerz for pointing out that `Array#wrap` is `active_support`s method.
It's implementation is simple, we can use it partially:
```ruby
def self.wrap(object)
  if object.nil?
    []
  elsif object.respond_to?(:to_ary)
    object.to_ary
  else
    [object]
  end
end
```

Comment (jonkerz) [reply lib/fortytwoish/client.rb]: http://route.github.io/2012/08/28/see-the-difference.html

Comment (lunich) [reply lib/fortytwoish/client.rb]: But still let's move to `initializer`?

Comment (davorb) [reply lib/fortytwoish/client.rb]: No, we can't do that. `Array.wrap` is only available in Rails. But `Array(numbers)` works.

Comment (davorb) [reply lib/fortytwoish/client.rb]: Done in babc851 (commit sha sounds like my last name)

**Review by jonkerz — APPROVED** [review]

**Review by lunich — APPROVED** [review]

**Review by sqbell — APPROVED** [review]

File: lib/fortytwoish/client.rb (hunk 1 above)
Comment (jonkerz) [reply lib/fortytwoish/client.rb]: > commit sha sounds like my last name

:smile: :smile: :smile:

**Pull request merged**